game_input.rb

```diff
@@ -0,0 +1,14 @@
+class GameInput
+
+    # rearranges a2 => 2a etc.
+    def reorder_move(move)
+        return move.chars.sort.join
+    end
```

Comment:
This is called `GameInput` yet the method here does no input. It is formatting output

Reply:
moved to game_validator